docs/modules/maintain-cluster/pages/rolling-upgrades.adoc

```diff
@@ -220,3 +253,14 @@ binary and algorithmic compatibility and some other steps.
 
 It is worth mentioning that a business app upgrade is orthogonal to a
 rolling member upgrade. A rolling business app upgrade may be done without upgrading the members.
+
+**Are features marked as Beta in my current version upgraded?**
```

Comment:
well, in theory they are upgraded. The important part about them is that features marked as Beta in a minor version are only supported in a cluster of the same minor version. So during upgrades, errors can happen. 

For example, if you upgrade compact from 5.2 -> 5.3 there are binary format changes, but only in some field types: Date Time etc. If the user does not use those fields, there'll be no fails. Otherwise, they'll fail. Also all the features that depend on SQL can fail in the same way.

but yes, after these fail, any new usages of the SQL will have the latest features available in the target release (GA version)

Reply:
I've reworded, let me know if it still needs work!